### DISCUSSION TITLE
Welcome to PMDM Discussions!

### DISCUSSION THREAD

**Discussion by Layne-Huang** [Layne-Huang/PMDM#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sergioo-herreraa** [comment]
@Layne-Huang Hi! I am having some trouble creating the first environment form your tutorial. 

![image](https://github.com/user-attachments/assets/ef8bb220-9276-4308-9399-05ebe25882c5)

I tried creating the environment with the original python version that my computer had (3.9), but I got a lot of conflicts and in the end the environment was not created. Then, I tried with the 2.7 version but I still see that there are some potential conflicts for the creation of the environment. Hence, I would like to ask you: what is the version of python that should be specified when creating these environments? Are there any other specifications that I should consider? 

I would really appreciate your help :).
